scripts/ci.sh

```diff
@@ -34,7 +34,8 @@ function main() {
               "${WORKSPACE}/tests/site.yml"
 
   # Sleep to allow Minecraft to boot.
-  sleep 5
+  # FIXME: A retry loop checking if it has launched yet would be better.
```

Comment:
Agreed. Maybe we can switch the CI script to Ansible as well and loop with `until` or `wait_for`.